src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -57,6 +58,9 @@
   {:render
    (fn [{:keys [props]}]
      [:div {:style {:textAlign "right"}}
+      [TopNavBarLink {:name "Import Data"
```

Comment:
That's a good idea. Something like:
```clojure
(def top-nav-bar-items
[{:key :workspaces
  :name "Workspaces"
  :component workspaces/Page}
  ...
]
```
should work. Feel free to go for it or just add a TODO for us to tackle.